kani-driver/src/main.rs

```diff
@@ -53,9 +53,10 @@ fn cargokani_main(input_args: Vec<OsString>) -> Result<()> {
     args.validate();
     let session = session::KaniSession::new(args.common_opts)?;
 
-    if matches!(args.command, Some(CargoKaniSubcommand::Assess)) || session.args.assess {
-        // Run cargo assess.
-        return assess::cargokani_assess_main(session);
+    if let Some(CargoKaniSubcommand::Assess(args)) = args.command {
+        return assess::cargokani_assess_main(session, args);
+    } else if session.args.assess {
```

Comment:
This was a hack to let us put something in compiletests for the original assess.

Reply:
Are you planning to add support to sub-commands in the manifest file?

Reply:
Probably not (I don't see why customers would want to add subcommands to a `Cargo.toml`). I think the route to get rid of this would be for a more flexible compiletest mode that can run arbitrary commands.

Reply:
I opened https://github.com/model-checking/kani/issues/2059 to track this.